rust/extractor/src/translate/base.rs

```diff
@@ -189,14 +189,16 @@ impl<'a> Translator<'a> {
             start.col + 1,
             &error_message
         );
-        let location = self.trap.emit_location_label(self.label, start, end);
-        self.trap.emit_diagnostic(
-            severity,
-            error_tag,
-            error_message,
-            full_error_message,
-            location,
-        );
+        if severity != DiagnosticSeverity::Debug {
```

Comment:
Perhaps `>` instead of `!=` ? I think it is unlikely we'll add levels lower than debug, but just in case.